### ISSUE TITLE
,WCAG 1.3.1,WCAG 4.1.2,Best Practice: Ensures role attribute has an appropriate value for the element (img[role="presentation"])

### ISSUE THREAD

**Issue by joosep41** [joosep41/Manguarvustus#1]
#### Issue
`Ensures role attribute has an appropriate value for the element` ([aria-allowed-role](https://dequeuniversity.com/rules/axe/4.4/aria-allowed-role?application=msftAI))
#### Target application
[Accessibility Fails](file:///C:/Users/j/dev/accessibility-fails/index.html)
#### Element path
img[role="presentation"]
#### Snippet
`<img role="presentation" src="images/decoration.png" width="368" height="118" alt="Decorative line clipart">`
#### How to fix
    Fix any of the following:
      ARIA role presentation is not allowed for given element
#### Environment
Chrome version 109.0.0.0
This accessibility issue was found using Accessibility Insights for Web 2.36.1 (axe-core 4.4.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
